### PR TITLE
Triple stream samples

### PR DESCRIPTION
**Pull request by panxiaojun233**

### CODE REVIEW

File: dubbo-samples-triple/src/main/java/com/apache/dubbo/sample/testing/ITestService.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.apache.dubbo.sample.testing;
```
Comment (AlbumenJ) [inline dubbo-samples-triple/src/main/java/com/apache/dubbo/sample/testing/ITestService.java:1]: Please add ASF license header

File: dubbo-samples-triple/src/main/java/com/apache/dubbo/sample/testing/TestServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,356 @@
+package com.apache.dubbo.sample.testing;
```
Comment (AlbumenJ) [inline dubbo-samples-triple/src/main/java/com/apache/dubbo/sample/testing/TestServiceImpl.java:1]: Please add ASF license header

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

**Comment by plusmancn** [comment]
Duplicate of #286 

It cloud be closed.

**Comment by plusmancn** [comment]
Duplicate of #356 

It could be closed.

**Comment by plusmancn** [comment]
Duplicate of #356

@AlbumenJ @guohao It could be closed.

**Pull request closed without merging**